src/angular-core/scrolling/BUILD.bazel

```diff
@@ -41,8 +41,8 @@ ng_test_library(
 ng_web_test_suite(
     name = "unit_tests",
     srcs = [
-        "@npm//:node_modules/resize-observer-polyfill/dist/ResizeObserver.js",
         "@npm//:node_modules/perfect-scrollbar/dist/perfect-scrollbar.js",
+        "@npm//:node_modules/resize-observer-polyfill/dist/ResizeObserver.js",
```

Comment:
This file should not be changed.
I will fix this the schematics in the future, but for now don't commit changes to this file.